library/src/main/res/values/strings.xml

```diff
@@ -70,4 +70,6 @@
     <string name="chucker_graphql_operation_is_empty"> &lt;Unable to discover GraphQL operation name&gt;</string>
     <string name="chucker_scroll_buttons_for_search">Buttons to scroll the search items</string>
     <string name="chucker_search_results_title">Search Results:</string>
+    <string name="chucker_copy_response">Copy Response</string>
+    <string name="chucker_response_copied">Response Copied</string>
```

Comment:
Those should be adapted for request/response

Reply:
changes done